common/dynamicconfig/constants.go

```diff
@@ -195,10 +195,24 @@ const (
 	// ReachabilityQueryBuildIdLimit limits the number of build ids that can be requested in a single call to the
 	// GetWorkerTaskReachability API.
 	ReachabilityQueryBuildIdLimit = "limit.reachabilityQueryBuildIds"
+	// ReachabilityQuerySetDurationSinceDefault is the minimum period since a version set was demoted from being the
```

Comment:
the two configs represent the same concept, right? why not collapse them?

Reply:
Removing a build id is more severe, I think we want different configs for them.

Reply:
ok, that's fair